Automattic/jetpack
# Sharing: update Twitter and X buttons to support official X button

See #34459

## Proposed changes:

- The official Twitter button is no more; it has been updated to the X design. Let's update our preview Twitter button match.
- Since we now have an official X button, let's offer it with the X service.

<img width="585" alt="Screenshot 2023-12-19 at 18 38 42" src="https://github.com/Automattic/jetpack/assets/426388/821bae96-de2e-4287-b3da-88e6718e9f8e">
<img width="827" alt="Screenshot 2023-12-19 at 18 38 34" src="https://github.com/Automattic/jetpack/assets/426388/f6d9cb54-66fb-4854-8578-ae91581fdf42">

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Jetpack > Settings > Sharing
* Enable sharing buttons
* Go to Settings > Sharing in wp-admin.
* Add a Twitter and a X button.
* Pick the official sharing button style.
* Ensure the preview looks good.
* Save your changes.
* Ensure your changes look good on the frontend as well.
